app/models/journey.rb

```diff
@@ -59,4 +63,22 @@ class Journey < ApplicationRecord
            :cancelled?,
            :rejected?,
            to: :state_machine
+
+  def for_feed
+    {
```

Comment:
We should keep consistent with the other feeds so that we can update the relationships/attributes in one way/place:

https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/models/move.rb#L172-L174

Reply:
I agree 👍